docs/examples/import-definitions/rabbitmq.yaml

```diff
@@ -12,7 +12,7 @@ spec:
             containers:
             - name: rabbitmq
               volumeMounts:
-              - mountPath: /path/to/exported/definitions.json
+              - mountPath: /path/to/exported/ # filename left out intentionally
```

Comment:
Should we use a more sensible path here?
I know that this is just an example. However, as user I wouldn't know what to choose as path here.
Does `/etc/rabbitmq/` make sense here?

Reply:
`/etc/rabbitmq/definitions.json` sounds reasonable, this is a piece of configuration. We can also update the docs on rabbitmq.com but FWIW so far I haven't seen any complaints about this example.

Reply:
What I like about `/path/to/exported/` is that it's clearly an example/dummy value and it also JustWorks™. Given the path is a dummy value, I think a user would be inclined to change it to something more sensible. What worries me about `/etc/rabbitmq/definitions.json` is that it might induce a user to think that the definitions must be in `/etc/rabbitmq`, which is not true.

What do you think?

Reply:
Okay, let's leave it as is :)